jenkins-scripts/docker/ign_gazebo-compilation.bash

```diff
@@ -38,11 +38,23 @@ if [[ ${IGN_GAZEBO_MAJOR_VERSION} -eq 5 ]]; then
 
   export BUILD_IGN_GUI=true
   export IGN_GUI_MAJOR_VERSION=5
-  export IGN_GUI_BRANCH=main
+  export IGN_GUI_BRANCH=ign-gui5
```

Comment:
Many of these branches dont exist, we should use `main` for the latest branches

Reply:
The gazebodistro file can be used as reference https://github.com/ignition-tooling/gazebodistro/blob/master/collection-edifice.yaml